docs/user-guide/cli-installing-ssh2-package.md

```diff
@@ -0,0 +1,101 @@
+# Installing the ssh2 Package for Zowe CLI
+​
+This topic describes how to install the ssh2 package to support Zowe CLI.  
```

Comment:
This describes how to install the pre-requisites to enable optional features in the ssh2 package.